packages/native/examples/with-distinct-filed-prop/App.js

```diff
@@ -0,0 +1,214 @@
+import React, { useState, useRef } from 'react';
```

Comment:
@anik-ghosh-au7 You've created a new example with `distinctField` prop but how is it different from the `basic` demo visually? There should be some meaningful use of that data that you receive after using the new props. You can use it in `SearchComponent` and group the results by authors and display the `inner_hits` in UI. Get approval before making this change everywhere.